src/pyOpenMS/tests/unittests/test_file_io.py

```
@@ -0,0 +1,453 @@
+     XTandemInfile.setPrecursorMassToleranceMinus
+     XTandemInfile.getPrecursorMassToleranceMinus
+     XTandemInfile.setPrecursorErrorType
+     XTandemInfile.getPrecursorErrorType
+     XTandemInfile.setFragmentMassErrorUnit
+     XTandemInfile.getFragmentMassErrorUnit
+     XTandemInfile.setPrecursorMassErrorUnit
+     XTandemInfile.getPrecursorMassErrorUnit
+    """
+    f = pyopenms.XTandemInfile()
+
+    f.setFragmentMassTolerance is not None
```

Ruff (0.14.10)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)